### PR TITLE
Updating Sdk version from preview to non-preview

### PR DESCRIPTION
**Pull request by anusapan**
## Description
The service has been in GA now.
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/DeviceProvisioningServices/DeviceProvisioningServices.Tests/DeviceProvisioningServices.Tests.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <PackageId>DeviceProvisioningServices.Tests</PackageId>
     <Description>DeviceProvisioningServices.Tests Class Library</Description>
     <AssemblyName>DeviceProvisioningServices.Tests</AssemblyName>
-    <VersionPrefix>1.0.1-preview</VersionPrefix>    
+    <VersionPrefix>1.0.1</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/DeviceProvisioningServices/DeviceProvisioningServices.Tests/DeviceProvisioningServices.Tests.csproj]: Tests are never published, we recommend simply setting it to `1.0.0` and never change that.

File: src/SDKs/DeviceProvisioningServices/Management.DeviceProvisioningServices/Microsoft.Azure.Management.DeviceProvisioningServices.csproj [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <Description>Provides management capabilities for Microsoft Azure IoT Device Provisioning Service.</Description>
     <AssemblyTitle>Microsoft Azure Provisioning Service Management</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.DeviceProvisioningServices</AssemblyName>
-    <Version>0.9.0-preview</Version>
+    <Version>0.9.0</Version>
```
Comment (dsgouda) [inline src/SDKs/DeviceProvisioningServices/Management.DeviceProvisioningServices/Microsoft.Azure.Management.DeviceProvisioningServices.csproj]: If you are planning to release a stable version, we recommend starting versioning from `1.0.0`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Left a couple of comments.

File: src/SDKs/DeviceProvisioningServices/Management.DeviceProvisioningServices/Microsoft.Azure.Management.DeviceProvisioningServices.csproj (hunk 2 above)
Comment (anusapan) [reply src/SDKs/DeviceProvisioningServices/Management.DeviceProvisioningServices/Microsoft.Azure.Management.DeviceProvisioningServices.csproj]: Done

File: src/SDKs/DeviceProvisioningServices/DeviceProvisioningServices.Tests/DeviceProvisioningServices.Tests.csproj (hunk 1 above)
Comment (anusapan) [reply src/SDKs/DeviceProvisioningServices/DeviceProvisioningServices.Tests/DeviceProvisioningServices.Tests.csproj]: Done

**Review by dsgouda — APPROVED** [review]
LGTM will merge on CIs passing

**Pull request merged**